api/python/cellxgene_census/release_process.md

```diff
@@ -112,6 +112,6 @@ To publish built release assets to PyPi (_note_: this will require your pypi/tes
    pipx run twine upload ./artifact/*
 6. Test the installation from PyPi, as a final sanity check. Note that it may take a minute for the new release to be visible on pypi.org:
    ```shell
-   pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple cellxgene-census
+   pip install -i cellxgene-census
```

Comment:
What is the `-i`?  I think perhaps a left over from the typo?

it is also sometimes necessary to add the arg `--no-cache-dir` to force it to hit pypi.org.  Same for the previous step.